node-server/package.json

```diff
@@ -6,18 +6,20 @@
   "scripts": {
     "start": "cross-env NODE_ENV=production node index.js",
     "dev": "cross-env NODE_ENV=development nodemon index.js",
-    "test": "cross-env NODE_ENV=test echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
```

Comment:
NODE_ENV is not set. Should probably be set as "cross-env NODE_ENV=test" ? And related to this, logger.info/error does not print anything if NODE_ENV is set to "test".